app/konnect/private-connections/aws-privatelink.md

```diff
@@ -0,0 +1,78 @@
+
+<li>Find the correct service name for your region in the tables below. Open the tab that matches your AWS region and use the PrivateLink service name for your {{site.konnect_short_name}} geo. For example, <code>com.amazonaws.vpce.us-east-2.vpce-svc-096fe7ba54ebc32db</code> for the us-east-2 AWS region and US {{site.konnect_short_name}} geo.
+
+{% navtabs %}
+
+{% navtab ap-southeast-2 %}
+| {{site.konnect_short_name}} Geographical Region | PrivateLink service name | Private DNS name
+| ------ | ------------ |
+| AP | com.amazonaws.vpce.ap-southeast-2.vpce-svc-05526f43d36cc10e1 | ap.svc.konghq.com
+| EU | com.amazonaws.vpce.ap-southeast-2.vpce-svc-02392dc520d2d9ad9 | eu.svc.konghq.com
+| ME | com.amazonaws.vpce.ap-southeast-2.vpce-svc-013380defe9ad33b6 | me.svc.konghq.com
+| US | com.amazonaws.vpce.ap-southeast-2.vpce-svc-0d986920a41ab0b80 | us.svc.konghq.com
+{% endnavtab %}
+
+{% navtab eu-central-1 %}
+| {{site.konnect_short_name}} Geographical Region | PrivateLink service name | Private DNS name
+| ------ | ------------ |
+| AP | com.amazonaws.vpce.eu-central-1.vpce-svc-0c3f0574080bdd859 | ap.svc.konghq.com
+| EU | com.amazonaws.vpce.eu-central-1.vpce-svc-05e6822fbce58e1a0 | eu.svc.konghq.com
+| ME | com.amazonaws.vpce.eu-central-1.vpce-svc-0e6497e6df9928a80 | me.svc.konghq.com
+| US | com.amazonaws.vpce.eu-central-1.vpce-svc-01d3dd232e277feeb | us.svc.konghq.com
+{% endnavtab %}
+
+{% navtab us-east-2 %}
+| {{site.konnect_short_name}} Geographical Region | PrivateLink service name | Private DNS name
+| ------ | ------------ |
+| AP | com.amazonaws.vpce.us-east-2.vpce-svc-03da89378358921bc | ap.svc.konghq.com
+| EU | com.amazonaws.vpce.us-east-2.vpce-svc-0cb28c923823735ac | eu.svc.konghq.com
+| ME | com.amazonaws.vpce.us-east-2.vpce-svc-0f1c86fb6399d4fe5 | me.svc.konghq.com
+| US | com.amazonaws.vpce.us-east-2.vpce-svc-096fe7ba54ebc32db | us.svc.konghq.com
+{% endnavtab %}
+{% endnavtabs %}
+</li>
+
+<li>Verify the service. If the service name cannot be verified, contact <a href="https://support.konghq.com/support/s/">Kong Support</a>.</li>
+
+<li>Once the service name is verified, select the VPC, subnets, and security groups to associate with this endpoint. Make sure that:
+<ul>
+<li>The security group accepts inbound traffic on TCP port 443.</li>
+<li>The DNS name parameter in the additional settings is enabled.</li>
+</ul>
+</li>
+
+<li>Create the VPC endpoint and wait for its status to change to available. It is recommended to wait for 10 minutes before attempting to use it.</li>
+
+<li>Update your data plane configuration to connect to the control plane through the private connection. Use the private DNS name that matches your control plane geo in the tables above. For the US {{site.konnect_short_name}} geo, the updated Kong data plane configuration looks like this:
+<pre>
+cluster_control_plane = us.svc.konghq.com/cp/{cluster_prefix}
+cluster_server_name = us.svc.konghq.com
+</pre>
+</li>
+</ol>
```

Comment:
Here's my suggested edits (the code block will break the GH suggestion thingy, so I'll just post it here):

> 1. Update your data plane configuration to connect to the control plane through the private connection. Use the private DNS name that matches your control plane geo in the tables above. For the US {{site.konnect_short_name}} geo, the updated Kong data plane configuration looks like this:
>   ```
>   cluster_control_plane = us.svc.konghq.com/cp/{cluster_prefix}
>   cluster_server_name = us.svc.konghq.com
>   ```

Also, I'm not exactly sure where I'd put this config for Konnect. Is there a doc we could link or maybe someplace we could reference in the UI or a file?

Reply:
I missed that, it's in kong.conf, I added it. Thanks!